README.md

```diff
@@ -1,11 +1,11 @@
-# flat-manager
+ # flat-manager
```

Comment:
Unnecessary whitespace change?

Reply:
Yes. :-)

Reply:
I removed the white space change through squashing.